Its.Log.Monitoring/Telemetry.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Its.Log.Instrumentation;
+
+namespace Its.Log.Monitoring
+{
+    public class Telemetry
+    {
```

Comment:
Agreed. Decoupling Its.Log.Monitoring from Its.Log is the goal but the duplication only makes sense when that happens.